docs/source/ref/settings.rst

```diff
@@ -426,6 +426,13 @@ Thumbnailer class that will be used to generate thumbnails. Available options:
 ``pip install django-oscar[easy-thumbnails]``. Custom thumbnailer class (based on
 ``oscar.core.thumbnails.AbstractThumbnailer``) can be used as well.
 
+``OSCAR_THUMBNAIL_DEBUG``
+-------------------------
+
+Default: ``Same as 'DEBUG'``
```

Comment:
This renders oddly. I think it's better only to render the `DEBUG` bit as a code string.